src/System.Data.Odbc/src/System/Data/Odbc/OdbcException.cs

```diff
@@ -2,12 +2,13 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.Data.Common;
 using System.Runtime.Serialization;
 using System.Text;
 
 namespace System.Data.Odbc
 {
+    [Serializable]
+    [System.Runtime.CompilerServices.TypeForwardedFrom("System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
     public sealed class OdbcException : System.Data.Common.DbException
```

Comment:
Desktop has `    ODBC32.RETCODE _retcode;    // DO NOT REMOVE! only needed for serialization purposes, because Everett had it.`. I don't ow why, since the serializer should be ignoring the fields, I guess that was why you didn't port it.

Reply:
Exactly.